src/components/BooksForm.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
```

Comment:
- [ ] Same concept. Need to be moved to `src/containers`.

Reply:
Thanks for your suggestion. It is implemented.